src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -1269,6 +1269,18 @@ impl<T: AstInfo> AstDisplay for CreateMaterializedViewStatement<T> {
             f.write_node(cluster);
         }
 
+        if !self.non_null_assertions.is_empty() {
+            f.write_str(" WITH (");
+            for (i, nna) in self.non_null_assertions.iter().enumerate() {
+                f.write_str("ASSERT NOT NULL ");
+                f.write_node(nna);
+                if i + 1 != self.non_null_assertions.len() {
+                    f.write_str(", ")
```

Comment:
nit: you can use `&display::comma_separated`?

Reply:
yeah, but then I'd have to make a separate node just for `ASSERT NOT NULL <identifier>`,  which seems to require more code than just inlining the comma-separated logic here.

Reply:
cool 👍